cplusplus/zdw/BufferedInput.h

```diff
@@ -271,7 +243,15 @@ class BufferedInput
 		return false;
 	}
 
-private:
```

Comment:
I don't think we should make all the data members public.

Reply:
Yeah; that's a good call.  *sigh*.  Fixing.